integrations/cli/index.test.ts

```diff
@@ -491,7 +492,7 @@ test(
       'pages/nested/foo.jsx': 'content-["pages/nested/foo.jsx"] content-["BAD"]',
     },
   },
-  async ({ fs, exec }) => {
+  async ({ fs, exec, expect }) => {
```

Comment:
Due to the concurrency, we have to use the `expect` provided by the test itself instead of the globally imported `expect`.